fedora/README.md

```diff
@@ -38,7 +38,7 @@ Documentation for this image is stored in the [`fedora/` directory](https://gith
 
 ## Issues
 
-If you have any problems with or questions about this image, please contact us by filing a bug on [Fedora's bugzilla page](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora) (choose `docker-io` as component and include details about image problems in the description) or through a [GitHub issue](https://github.com/lsm5/docker-brew-fedora/issues).
+If you have any problems with or questions about this image, please file a ticket with [Fedora Release Engineering](https://fedorahosted.org/rel-eng/newticket) for `fedora:22` and `fedora:21` or file a [GitHub issue](https://github.com/fedora-cloud/docker-brew-fedora/issues/new) for `fedora:rawhide` and `fedora:20`.
```

Comment:
This URL needs to change in https://github.com/docker-library/docs/blob/f68b2f970b50ae0af376a43d3bd03de59cd99d4e/update.sh#L34, and the rest needs to change in either the existing `mailing-list.md`, or in a new `user-feedback.md`.